server/src/main/java/online/dipa/hub/convert/MilestoneTemplateToMilestoneConverter.java

```diff
@@ -12,9 +12,20 @@
 public class MilestoneTemplateToMilestoneConverter implements Converter<MilestoneTemplateEntity, Milestone> {
     @Override
     public Milestone convert(final MilestoneTemplateEntity templateEntity) {
-        return new Milestone().id(templateEntity.getId())
-                              .name(templateEntity.getName())
-                              .date(LocalDate.now()
-                                             .plusDays(templateEntity.getDateOffset()));
+        final Milestone milestone = new Milestone().id(templateEntity.getId())
+                                                   .name(templateEntity.getName())
+                                                   .date(LocalDate.now()
+                                                                  .plusDays(templateEntity.getDateOffset()));
+        switch (templateEntity.getStatus()) {
+        case "offen":
+            milestone.status(Milestone.StatusEnum.OFFEN);
+            break;
+        case "erledigt":
+            milestone.status(Milestone.StatusEnum.ERLEDIGT);
+            break;
+        }

```

Comment:
Bitte keine Strings in switch-Anweisungen oder ähnlichem verwenden. Ich denke das kann dadurch ersetzt werden:
`.status(Milestone.StatusEnum.fromValue(templateEntity.getStatus()));`